src/actions/profile.test.js

```diff
@@ -7,6 +11,24 @@ import {
   saveUserProfile,
 } from './profile';
 
+describe('editable field actions', () => {
+  it('should create an open action', () => {
+    const expectedAction = {
+      type: EDITABLE_FIELD_OPEN,
+      fieldName: 'name',
+    };
+    expect(openEditableField('name')).toEqual(expectedAction);
+  });
+
+  it('should create a closed action', () => {
+    const expectedAction = {
+      type: EDITABLE_FIELD_CLOSE,
+      fieldName: 'name',
+    };
+    expect(closeEditableField('name')).toEqual(expectedAction);
+  });
+});
```

Comment:
Are these types of tests useful? Seems like a lot of extra work/code over time when many action creators will look like this.

```
export const openEditableField = fieldName => ({
  type: EDITABLE_FIELD_OPEN,
  fieldName,
});
```

Reply:
Yup.  It tells us that fieldName was properly added to the object, and that we have the right constant in there.

When you add a dozen actions all at once with similar names and/or a dozen arguments, it's nice to double-check they're correct. :)

Reply:
Furthermore, code without tests is much less likely to get tested when modified than code with existing tests.  It may seem a bit silly now, but we're helping people do the right thing later, too.

Reply:
👍